app/src/main/res/values/strings.xml

```diff
@@ -55,7 +55,7 @@
     <!-- Title for the button which navigates the user to show all of their saved bookmarks. -->
     <string name="recently_saved_show_all">Show all</string>
     <!-- Content description for the button which navigates the user to show all of their saved bookmarks. -->
-    <string name="recently_saved_show_all_content_description">Show all saved bookmarks button</string>
+    <string name="recently_saved_show_all_content_description2">Show all saved bookmarks</string>
```

Comment:
We have to keep the old strings and tag it for removal. See https://github.com/mozilla-mobile/fenix/wiki/Removing-strings

Can change the suffix to `_2`?
```suggestion
    <string name="recently_saved_show_all_content_description_2">Show all saved bookmarks</string>
```

Reply:
Thank you for the link, I was not aware the procedure has changed. It seems a bit overcomplicated. 
Maybe if the automation part for updating strings that need a translation in Pontoon would be improved we could avoid jumping through these hoops on Fenix / Focus.